source/contrib/automated_contributors.py

```diff
@@ -0,0 +1,47 @@
+from github import Github
+from github import Auth
+
+access_token = "YOUR_ACCESS_TOKEN"
+
+auth = Auth.Token(access_token)
+
+g = Github(auth=auth)
+
+repositories = [
+    "openchemistry/avogadrolibs",
+    "openchemistry/avogadroapp",
+    "openchemistry/fragments",
+    "openchemistry/molecules",
+    "openchemistry/crystals",
+    "openchemistry/avogenerators",
+    "openchemistry/avogadro-commands",
+    "openchemistry/avogadro-cclib",
+    "cryos/avogadro",
```

Comment:
The cryos/avogadro repository is now closed, so I moved the contributors to a separate list already. That way we don't continue to query for the same response.